code/game/objects/items/mop.dm

```diff
@@ -52,7 +52,7 @@
 		return CLEAN_BLOCKED
 	if(reagents.has_reagent(amount = 1, chemical_flags = REAGENT_CLEANS))
 		return CLEAN_ALLOWED
-	return CLEAN_BLOCKED|CLEAN_NO_XP
+	return CLEAN_ALLOWED|CLEAN_NO_XP
```

Comment:
This will allow cleaning with any reagent, including blood and alike - maybe make a new flag which gets passed into ``clean()`` on the component to continue the interaction but not actually remove the decals? Otherwise this is functionally the same as having any ``REAGENT_CLEANS`` reagent, just without the XP gain